feature_engine/variable_manipulation.py

```diff
@@ -2,14 +2,14 @@
 # License: BSD 3 clause
 # functions shared across transformers
 
-from typing import List, Optional, Union
+from typing import List, Optional, Union, Any
 
 import pandas as pd
 
+Variables = Union[None, int, str, List[Union[str, int]]]
```

Comment:
@solegalli note that you can use typehint aliases like this (bit neater) https://docs.python.org/3/library/typing.html#type-aliases